docs/user-guide/quickstart.md

```diff
@@ -57,6 +57,9 @@ E.g. To fetch the project with name `'Number 1'`:
 ```python
 project = tamr.projects.by_name('Number 1')
 ```
+``` note::
+    If working with projects over multiple instances, consider using ``by_external_id``.
```

Comment:
We should emphasize that this is only a concern for working *across* instances and only for migrations/promotions.

```suggestion
    If working with projects over across Tamr instances for migrations or promotions, use external IDs ( via ``by_external_id``) instead of name (via ``by_name``).
```